ArcGISRuntimeSDKQt_CppSamples/Maps/ShowLocationHistory/ShowLocationHistory.cpp

```diff
@@ -0,0 +1,131 @@
861413,4040315.1834499966), Point(-13007031.564306224,4040541.39735479), Point(-13007057.042322807,4040339.117344355), Point(-13007011.490717405,4040449.52208287), Point(-13006952.428042606,4040544.4855992175), Point(-13006995.277434127,4040379.2645219946), Point(-13006946.637584293,4040226.7824530574), Point(-13006908.806589976,4040240.2935224567), Point(-13006901.472009445,4040303.2165028006), Point(-13006867.501320671,4040238.363369685), Point(-13006879.854298407,4040164.245503266), Point(-13006821.563684715,4040195.1279476066), Point(-13006794.541545916,4040224.8523002844), Point(-13006779.4863543,4040166.5616865917), Point(-13006688.769174052,4040199.374283703), Point(-13006678.732379641,4040249.5582557567), Point(-13006600.75420769,4040344.5217721006), Point(-13006478.768552545,4040397.4079580316), Point(-13006460.625116494,4040465.3493355783), Point(-13006471.433972025,4040525.956132594), Point(-13006449.044199886,4040656.0484293764), Point(-13006472.592063695,4040680.7543848488), Point(-13006468.345727604,4040742.133242973), Point(-13006504.246569151,4040809.302559414), Point(-13006465.643513724,4040807.372406643), Point(-13006454.44862765,4040841.7291259686), Point(-13006497.298019174,4040861.416684236), Point(-13006470.66191093,4040884.9645480453), Point(-13006528.952524621,4040927.0418784595), Point(-13006488.805346979,4040929.7440923396), Point(-13006467.187635941,4040970.6633310863), Point(-13006510.809088573,4040986.1045532566), Point(-13006483.400919221,4041011.1965392837), Point(-13006509.26496636,4041093.4210473364), Point(-13006495.367866406,4041110.0203611697), Point(-13006503.474508055,4041175.6455553896), Point(-13006600.754207738,4041320.4070132314), Point(-13006637.813140953,4041406.8778573833), Point(-13006633.566804856,4041479.8376321383), Point(-13006606.158635503,4041526.9333597533), Point(-13006582.224741139,4041544.6907652495), Point(-13006537.05916629,4041570.554812385), Point(-13006571.801916175,4041578.6614540224), Point(-13006598.051993866,4041633.4777927273), Point(-13006550.57023569,4041631.161609402), Point(-13006480.69870537,4041677.871306458), Point(-13006382.26091403,4041783.2576477686), Point(-13006354.466714123,4041873.588797465), Point(-13006273.014267175,4041992.872238723), Point(-13006138.675634298,4042124.1226271675), Point(-13006090.421815023,4042234.527365678), Point(-13006031.359140223,4042273.9024822125), Point(-13005898.178598998,4042420.98012338), Point(-13005806.689357642,4042428.314703906), Point(-13005670.420571996,4042393.957984575), Point(-13005583.177666733,4042405.924931757), Point(-13005500.567128133,4042402.064626215), Point(-13005475.475142105,4042385.8513429365), Point(-13005446.522850536,4042364.2336318977), Point(-13005417.570558969,4042337.2114930963), Point(-13005398.269031256,4042312.891568178), Point(-13005336.504142575,4042238.3876712057), Point(-13005192.514745843,4042221.0162962615), Point(-13005034.628249153,4042147.6704909513), Point(-13004884.076332992,4042148.8285826137)};
```

Comment:
we should wrap this line I think

Reply:
Is this from the Polyline I sent you? That was only a suggestion - feel free to create a polyline for somewhere unique.

Also, instead of creating a QList of Points, I believe you can do a Polyline::fromJson() and pass in Geometry JSON. Not required but might be a little simpler.

Reply:
The polyline could then be directly fed into the SimulatedLocationDisplay with no need for the builder on line 67